src/jsonToMarkdown.tsx

```diff
@@ -0,0 +1,204 @@
+  // }, underline is not supported in markdown
+  'strikethrough': (child: any, value: any) => {
+    return `~~${child}~~`;
+  },
+  'superscript': (child: any, value: any) => {
+    return `^${child}^`;
+  },
+  'subscript': (child: any, value: any) => {
+    return `~${child}~`;
+  },
+  'inlineCode': (child: any, value: any) => {
+    return `\`${child}\``
+  },
+}
+
+const getOLOrULStringFromJson = (value: any) => {
+  let child = ''
+  let nestedListFound = false
+  if(value.type === 'ol'){
+  let start = parseInt(value?.attrs?.start || 1)
+  Array.from(value.children).forEach((val: any, index) => { 
+    if(val.hasOwnProperty('type') && val.type === 'li' && value.children[index + 1] && value.children[index + 1]?.type === 'ol'){
+      let liChildren = jsonToMarkdownSerializer(val)
+      let nestedListChildren = getOLOrULStringFromJson(value.children[index + 1])
+      let indentedNestedListChildren = nestedListChildren.split('\n').filter((child) => child.length).map((child) => `  ${child}`).join('\n')
+      child += `${index + start}. ${liChildren}\n${indentedNestedListChildren}\n`
+      nestedListFound = true
+    }
+    else if(val.hasOwnProperty('type') && val.type !== 'ol'){
+      let liChildren = jsonToMarkdownSerializer(val) 
+      child += `${nestedListFound ? (index + start - 1): index + start}. ${liChildren}\n`
+    }
+
+  })
+  }
+  if(value.type === 'ul') {
+    let symbol = value?.attrs?.listStyleType || '- '
+    Array.from(value.children).forEach((val: any, index) => {
+      if(val.hasOwnProperty('type') && val.type === 'li' && value.children[index + 1] && value.children[index + 1]?.type === 'ol'){
+        let liChildren = jsonToMarkdownSerializer(val)
+        let nestedListChildren = getOLOrULStringFromJson(value.children[index + 1])
+        let indentedNestedListChildren = nestedListChildren.split('\n').filter((child) => child.length).map((child) => `  ${child}`).join('\n')
+        child += `${symbol}${liChildren}\n${indentedNestedListChildren}\n`
+      }
+      else if(val.hasOwnProperty('type') && val.type !== 'ol'){
+        let liChildren = jsonToMarkdownSerializer(val) 
+        child += `${symbol}${liChildren}\n`
+      }
+    })
+  }
+  return `
+${child}`
+}
+
+export const jsonToMarkdownSerializer = (jsonValue: any): string => {
+  if (jsonValue.hasOwnProperty('text')) {
+    let text = jsonValue['text'].replace(/</g, '&lt;').replace(/>/g, '&gt;')
+    if (jsonValue['break']) {
+      text += `<br/>`
+    }
+    if (jsonValue.text.includes('\n') && !jsonValue['break']) {
+      text = text.replace(/\n/g, '<br/>')
+    }
+    Object.entries(jsonValue).forEach(([key, value]) => {
+      if (TEXT_WRAPPERS.hasOwnProperty(key)) {
+        text = TEXT_WRAPPERS[key](text, value)
+      }
+    })
+    return text
+  }
+  let children: any = ''
+  if (jsonValue.children) {
+    children = Array.from(jsonValue.children).map((child) => jsonToMarkdownSerializer(child))
+    if (jsonValue['type'] === 'blockquote') {
```

Comment:
@Ashwini-Rathod There's no error handling for unexpected jsonValue['type'] values. It would be good to have a default case that logs an error or throws an exception.